pkg/traceability/sampling/globalsampling.go

```diff
@@ -14,6 +14,7 @@ var agentSamples *sample
 type Sampling struct {
 	Percentage      int  `config:"percentage"    validate:"min=0, max=100"`
 	PerAPI          bool `config:"per_api"`
+	PerSub          bool `config:"per_api"`
```

Comment:
config setting needs updated

Reply:
What about it needs to be updated?

Reply:
PerSub bool `config:"per_api"`. -> PerSub bool `config:"per_subscription"`